cpp/ql/src/Best Practices/Likely Errors/CommaBeforeMisleadingIndentation.qhelp

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+If the expression to the right of a comma operator starts at an earlier column than the expression to the left, then
+this suspicious indentation possibly indicates a logic error, caused by a typo that may escape visual inspection.
```

Comment:
I'm not entirely sure what we're trying to say here. Is this only a problem when we have a comma and bad indentation together? Or, is a comma always a problem, but we're only flagging code where the indentation suggests that it is the comma that's wrong, rather than the indentation?

Reply:
The issue is about bad indentation following a comma, so it requires both of those elements.  Perhaps the confusing part here is actually the use of "left" and "right" to describe relative positions when at least one of these expressions occurs on a different line (with indentation).
```suggestion
If the expression after the comma operator starts at an earlier column than the expression before the comma, then
this suspicious indentation possibly indicates a logic error, caused by a typo that may escape visual inspection.
```